src/debug_commads.c

```diff
@@ -207,7 +207,7 @@ DEBUG_COMMAND(DumpNumericIndex) {
   while ((currNode = NumericRangeTreeIterator_Next(iter))) {
     NumericRange *range = currNode->range;
     if (range) {
-      IndexReader *reader = NewNumericReader(NULL, range->entries, NULL, range->minVal, range->maxVal);
+      IndexReader *reader = NewNumericReader(NULL, range->entries, NULL, range->minVal, range->maxVal, true);
```

Comment:
Maybe we can create a flag here, looking at the code it is not clear what `true` means and we need to jump to function definition to understand better.

Reply:
Unless you insist, I would leave it as-is - also `NULL` is unclear and requires looking at the definition, especially for functions with 5 or 6 args.